ReactWindows/ReactNative/Views/TextInput/ReactTextInputManager.cs

```diff
@@ -212,6 +212,17 @@ public void SetPlaceholder(ReactTextBox view, string placeholder)
         }
 
         /// <summary>
+        /// Sets the background color for the <see cref="ReactTextBox"/>.
+        /// </summary>
+        /// <param name="view">The view instance.</param>
+        /// <param name="color">The masked color value.</param>
+        [ReactProp(ViewProps.BackgroundColor, CustomType = "Color")]
+        public void SetBackgroundColor(ReactTextBox view, uint color)
```

Comment:
I think this may need to be nullable (i.e., uint?).  You'll want to figure out what the correct behavior is if the backgroundColor is set back to null.